### PR TITLE
Doc feature: Release Detail

### LINKED ISSUE #255: Document application capabilities
We need to document current capabilities of application, with good screenshots, to let new and old users know of all features. No need to write too much of text, we rather need discovery walkthrough.

Probably, it should be a separate Markdown page, or even multiple pages one per area.

Below is the checklist of all the capabilities, without particular sorting.

----

# General layout
~~Shutting down the app~~
REST API docs
Reset cache
~~Multi-cluster~~

# Installed releases list
~~Filter by namespaces and free text~~
~~List of releases - with release status+installed resources status.~~
~~Basic info on release: chart, version, icon etc.~~
Indication of upgrade possible/repo suggested. 

# Release details
 Revision history - status, age, upgrade or downgrade, rollback icon
 Uninstalling
 Running test hooks
 Update repo / add missing repo
 Upgrade / Reconfiguring / Even without source chart
 Rollback
 Release revision details
  Basic info - revision, date, message, cluster, status
  Resources tab - kind, name, status, msg, describe, scan
  Manifests tab - text, diff with previous, diff with specific
  Values tab - text, diff, + user-defined flag
  Notes - text, diff

# Install/Upgrade/Reconfigure dialog
 Version selector - with originating repo info, current version displayed and marked
 Release name, namespace, cluster
 Values editor
 Values reference doc
 Changes diff preview
 Problem scanning btn

# Repo list
  - [local] pseudo-repo
  - note on autoupdate repo

~~updating repo~~
~~add new repo~~
~~delete repo~~
~~triggering install~~

### PR DESCRIPTION
**Pull request by pushker001**
Help users to know the capability of Helm-Dashboard

- [X] The title of my pull request is a short description of the changes
- [X] This PR relates to some issue:  #255

### CODE REVIEW

**Comment by undera** [comment]
Note how I edited the description of PR. It does not close issue #255, but it relates to it. This way, we'll have all the PRs linked to the same issue, and the issue is still open.

File: FEATURES.md [hunk 1]
```diff
@@ -46,4 +46,12 @@ Namespaces are a way to organize clusters into virtual sub-clusters — they can
 ![Releases4](images/screenshot_release4.png)
 
 Updated" refers to the amount of time that has passed since the last revision of the release. Whenever you install or upgrade the release, a new revision is created. You can think of it as the "age" of the latest revision.
-![Releases5](images/screenshot_release5.png)
\ No newline at end of file
+![Releases5](images/screenshot_release5.png)
+
+# Release details
```
Comment (undera) [inline FEATURES.md:51]: It's good for the start. 

I would mention the block on the left is called "Revision History". It displays the list of past release revisions, with the current revision on top. And each item in it shows status, age and version upgrade/downgrade.

File: FEATURES.md [hunk 2]
```diff
@@ -46,4 +46,12 @@ Namespaces are a way to organize clusters into virtual sub-clusters — they can
...
+This indicates the status of the deployed release, and 'Age' represents the amount of time that has passed since the creation of the revision until now.
+![Detail](images/screenshot_release_detail.png)
+
+You can use the Upgrade/Downgrade button to switch to different release versions, as shown in the figure.
```
Comment (undera) [inline FEATURES.md:55]: This specific button is tricky, it changes depending on various situations. Current description is good for the start, but there should be several more variants described, as well as other buttons. This whole button block deserves description as "Action Buttons" subsection.

File: FEATURES.md (hunk 1 above)
Comment (undera) [reply FEATURES.md:51]: Also, the screenshot of screenshot_release_detail1.png has much better revision history illustration.

**Comment by undera** [comment]
Please see my comments. I can accept PR as it is now, with future improvements. If you prefer improving current PR - let me know.

**Comment by pushker001** [comment]
> you prefer improving current PR - let me kno

yes

**Pull request merged**